src/System.Drawing.Common/tests/Helpers.cs

```diff
@@ -12,6 +12,7 @@ namespace System.Drawing
     public static class Helpers
     {
         public const string GdiplusIsAvailable = nameof(Helpers) + "." + nameof(GetGdiplusIsAvailable);
+        public const string IsNotLinuxAndGdiplusIsAvailable = nameof(Helpers) + "." + nameof(CheckIsNotLinuxAndGdiplusIsAvailable);
```

Comment:
Instead of adding this, you should just annotate the failing tests with `[ActiveIssue(20884, TestPlatforms.AnyUnix)]` like the others.